scripts/it.sh

```diff
@@ -7,4 +7,4 @@ cd "${script_full_path}"/.. || exit
 
 ./scripts/build-all.sh
 cd golem-cli
-RUST_LOG=info cargo test
+RUST_LOG=info cargo test --test integration
```

Comment:
maybe run sharding tests here too?

Reply:
It's quite unstable, but let's try any way.
Done.